src/lib/Libifl/pbs_ifl_wrap.c

```diff
@@ -5766,7 +5766,7 @@ SWIGINTERN PyObject *_wrap_pbs_statsched(PyObject *SWIGUNUSEDPARM(self), PyObjec
     SWIG_exception_fail(SWIG_ArgError(res3), "in method '" "pbs_statsched" "', argument " "3"" of type '" "char *""'");
   }
   arg3 = (char *)(buf3);
-  result = (struct batch_status *)pbs_statsched(arg1,PBS_DFLT_SCHED_NAME,arg2,arg3);
+  result = (struct batch_status *)pbs_statsched(arg1,arg2,arg3);
```

Comment:
same nit here, please add spaces after commas

Reply:
I think we can ignore this file as it is auto-generated. I edited it by following the mutlit sched changes